qemu-test-init

```diff
@@ -227,7 +224,8 @@ if [ -n "$INTERACTIVE_SHELL" ]; then
   sleep 1
 fi
 
-if meson test -C $BUILD_DIR "$MESON_TEST" -v; then
+eval "set -- $line"
+if meson test -C $BUILD_DIR "$@" -v; then
```

Comment:
Were is `$line` set? Perhaps it's easier to just avoid the quoting on $MESON_TEST? If there are spaces in the variable, meson should handle that fine.

Reply:
arf, it is "$test" :facepalm: it is remaining artifact of testing.

Sure, it is another possibility. It is not [shellcheck](https://www.shellcheck.net/wiki/SC2086) compatible.